src/types/form-element.ts

```diff
@@ -323,6 +323,13 @@ export interface SelectOptions {
    * @type {string}
    */
   getOptions?: string;
+
+  /**
+   * Allow Custom values
+   *
+   * @type {boolean}
+   */
+  allowCustomValues?: boolean;
```

Comment:
please rename to `allowCustomValue`

Reply:
please add migration and make the option is required or (remove it from the json if false)